api/weather.py

```diff
@@ -38,6 +40,35 @@
     "8000": "Thunderstorm",
 }
 
+conn = sqlite3.connect('/db/stats.db', check_same_thread=False)
+cur = conn.cursor()
+
+
+def weather_details(location):
+    weather_data: str
+    try:
+        params: Dict[str, str] = {
+            'location': f'{location.latitude},{location.longitude}',
+            'apikey': config["CLIMACELL_API_KEY"],
+            'fields': "temperature,humidity,windSpeed,weatherCode,particulateMatter25",
+        }
+
+        response = get('https://api.tomorrow.io/v4/timelines?', params=params).json()['data']
+        data: Dict = response['timelines'][0]['intervals'][0]['values']
+
+        conditions: str = data['weatherCode']
+        humidity: str = data['humidity']
+        pm25: str = data['particulateMatter25']
+        temperature: str = data['temperature']
+        wind_speed: str = data['windSpeed']
+
+        weather_data = f"*{location.address}*\n" \
```

Comment:
Non-urgent but if you could parse the address string to be not a address dump that would be great.